src/backwards/configuration.ts

```diff
@@ -0,0 +1,234 @@
+import cloneDeep from 'lodash.clonedeep';
+import {isObject} from "./utils/isObject";
+/**
+ * Class for defining account configuration options.
+ * Depends on 'utils'
+ */
+
+
+/**
+ * Assign values from sources if they are not defined in the destination.
+ * Once a value is set it does not change
+ * @function Util.defaults
+ * @param {Object} destination - the object to assign defaults to
+ * @param sources
+ * @param {...Object} source - the source object(s) to assign defaults from
+ * @return {Object} destination after it was modified
+ */
+const defaults = (destination:{}, ...sources: object[])=>{
```

Comment:
I know this is not a refactoring task, but can we please(!!) find a better name to this thing?

The name is meaningless.
`setDefaultValue`? `useDefaultValues`?